tests/demo_empty_suffix/hello

```diff
@@ -0,0 +1 @@
+hello {{ name }}!
```

Comment:
Tests not ready, could you show me again how to generate them from the tests directly rather than creating `demo` folders? :slightly_smiling_face:

Reply:
Yes, see for example https://github.com/copier-org/copier/blob/1f24b5a02e33960cd7d71c998e475b253efd62ae/tests/test_tasks.py#L9